.circleci/config.yml

```diff
@@ -200,30 +206,41 @@ commands:
             echo "RELEASE_VERSION is: ${RELEASE_VERSION}"
 
   restore_dependency_cache:
-    # This logic cannot be done within the node orb currently. This is due to the fact that we use
-    # the package.json instead of the package-lock.json for our checksum.
     parameters:
       base-cache-path:
         type: string
         default: << pipeline.parameters.cache-version >>
     steps:
       - restore_cache:
           keys:
-            - << parameters.base-cache-path >>{{ checksum "package.json" }}
+            - << parameters.base-cache-path >>{{ checksum "package-lock.json" }}
```

Comment:
so we were trigging caches off the package.json file vs package-lock.json.  I'm pretty sure this was a remnant of when we didn't have a package-lock.json file and didn't make sense.  I updated it to base the cache off the package-lock.json file.